RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate DisplayPrefsView to Typescript

## Proposed changes

Migrate DisplayPrefsView to TS

## Issue(s)	


## How to test or reproduce

- Click on `hamburguer` icon, on the SideBar click on `Display`, then change the preferences of display and back to list of rooms to check the difference

## Screenshots

## Types of changes




## Checklist



## Further comments
